.buildkite/pipelines/pull_request/security_solution.yml

```diff
@@ -48,6 +48,8 @@ steps:
     depends_on: build
     timeout_in_minutes: 120
     parallelism: 1
+    retry:
+      automatic: false
```

Comment:
I understand we are disabling this because we are considering with the Cypress retries are enough. That is correct?

Reply:
correct, we're going to re-enable it when we adjust all the tests to support "burn"

Reply:
great!! let's communicate and socialize this change as well as the new way of using tags to avoid misunderstandings inside teams :)